source/deployment/bulk-loading-about.rst

```diff
@@ -0,0 +1,19 @@
+.. _about-bulk-loading-command:
+
+About the bulk loading command
+==============================
+
+The bulk loading command is interruptible and idempotent
```

Comment:
"...idempotent" 

I'm not sure who's going to be reading this particular doc but, as a non-developer, I did not know what this word meant and had to look it up...

Meaning: "denoting an element of a set which is unchanged in value when multiplied or otherwise operated on by itself"

Not sure whether we need to expand on the meaning of the word?